module/Calculator/Mileage.h

```diff
@@ -0,0 +1,52 @@
+/**
+ **  @file Mileage.h
+ **  @brief 走行距離を計算するクラス
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+ **  @author desty505
+=======
+ **  @author TanakaShinnosuke
+>>>>>>> work-4
+=======
+ **  @author TanakaShinnosuke
+>>>>>>> work-4
+=======
+ **  @author TanakaShinnosuke
+>>>>>>> work-4
+ **/
```

Comment:
```suggestion
 **  @author desty505
```
コンフリクトの解消とかするときにも出てくると思うんだけど、この辺は普通に消しちゃって良いよ